main.go

```diff
@@ -2,17 +2,26 @@ package main
 
 import (
 	"app/base/core"
+	"app/base/utils"
 	"app/evaluator"
 	"app/listener"
 	"app/manager"
 	"app/vmaas_sync"
 	"log"
 	"os"
+	"runtime/debug"
 )
 
+func handlePanics() {
```

Comment:
Ok, but could we store this method somewhere in utils package? Can we write test for it?